libvast/test/transform.cpp

```diff
@@ -49,13 +68,33 @@ struct transforms_fixture {
     }
     return builder->finish();
   }
+  // Creates a table slice with a single string field and random data.
+  static std::pair<vast::table_slice, vast::table_slice>
+  make_proj_and_del_testdata(vast::table_slice_encoding encoding
+                             = vast::defaults::import::table_slice_type) {
+    auto builder = vast::factory<vast::table_slice_builder>::make(
+      encoding, testdata_layout2);
+    REQUIRE(builder);
+    auto builder2 = vast::factory<vast::table_slice_builder>::make(
+      encoding, testresult_layout2);
+    REQUIRE(builder2);
+    for (int i = 0; i < 10; ++i) {
+      auto uuid = vast::uuid::random();
+      auto str = fmt::format("{}", uuid);
+      auto str2 = fmt::format("test-datum {}", i);
+      auto str3 = fmt::format("note {}", i);
+      REQUIRE(builder->add(str, str2, vast::integer{i}, str3));
+      REQUIRE(builder2->add(str, vast::integer{i}));
+    }
+    return {builder->finish(), builder2->finish()};
+  }
 };
 
 FIXTURE_SCOPE(transform_tests, transforms_fixture)
 
 TEST(delete_ step) {
   auto slice = make_transforms_testdata();
-  vast::delete_step delete_step("uid");
+  vast::delete_step delete_step("uid"); // TODO:make it a list of what to delete
```

Comment:
Do you still intend to do this in the same PR? I think the change itself would be fairly trivial now.

Reply:
My plan is same PR, but different commit. Would it be better in another PR?